dbms/src/Storages/KVStore/MultiRaft/RegionData.h

```diff
@@ -62,7 +62,8 @@ class RegionData
 
     // Payload size in RegionData, show how much data flows in/out of the Region.
     size_t dataSize() const;
-    // `dataSize()` plus the decoded data cached.
+    // Reflects the number of bytes of memory currently occupied by this object.
```

Comment:
```suggestion
    // Reflects most of bytes of memory currently occupied by this object.
```

The are still some other structures like OrphanKeysInfo